src/WindowManager.vala

```diff
@@ -100,6 +100,11 @@ namespace Gala {
         private bool animating_switch_workspace = false;
         private GestureAnimationDirector gesture_animation_director;
 
+        /**
+         * Amount of pixels to move on the nudge animation.
+         */
+        public const float NUDGE_GAP = 32;
```

Comment:
Does this need to be multiplied by scale?

Reply:
I thought exactly the same, but it is not multiplied in the workspace switch animation (in `WindowManager.vala`), so I leave it the same way. However, I suspect it is a bug in the original implementation. Should I fix it in both places?

Reply:
That would be nice if it's not too much trouble :)

Reply:
Done :D